### ISSUE TITLE
Communication in X direction maybe broken for scalar build

### ISSUE THREAD

**Issue by diptorupd** [JeffersonLab/qphix#10]

**Comment by martin-ueding** [comment]
Scalar in the sense of no MPI? What happens that lets it appear broken?

**Comment by bjoo** [comment]
I think maybe in the sense of no vectorization? I think the scalar code doesn’t 
work in X (and we haven’t really suported scalar).

Best,
 B
> On Apr 29, 2017, at 9:20 AM, Martin Ueding <notifications@github.com> wrote:
> 
> Scalar in the sense of no MPI? What happens that lets it appear broken?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

--------------------------------------------------------------------------------------
Dr Balint Joo              High Performance Computational Scientist
Jefferson Lab
12000 Jefferson Ave, Suite 3, MS 12B2, Room F217,
Newport News, VA 23606, USA
Tel: +1-757-269-5339,      Fax: +1-757-269-5427
email: bjoo@jlab.org       
-------------------------------------------------------------------------------------

**Comment by ddkalamk** [comment]
Yes, originally Scalar code was added to allow running QPhiX code on BG/Q. We have some issue when running multinode with comms needed in X direction.
